src/stories/Inputs/Upload/Upload.test.tsx

```diff
@@ -10,7 +11,18 @@ const { Default } = composeStories(stories);
 
 describe('Upload unit tests', () => {
   it('Should render the Upload component', () => {
-    const { getByText } = render(<Default />);
-    expect(getByText('Upload Works!')).toBeInTheDocument();
+    const { getByTestId } = render(<Default />);
+    expect(getByTestId('file-input')).toBeInTheDocument();
+  });
+
+  it('Should be equal to the file that is selected', () => {
+    const testFile = new File(['Test File'], 'testFile.pdf', {
+      type: 'application/pdf',
+    });
+    const { getByTestId } = render(<Default />);
+    const fileInput = getByTestId('file-input') as HTMLFormElement;
+    userEvent.upload(fileInput, testFile);
```

Comment:
This second test is an example of us testing something that isn't just what's been rendered to the screen. We use the `userEvent` API to simulate adding a file to the `Upload` input. Then we can write assertions about what the input should equal once the file has been added. 

For more on how the `file` type input work see [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file).